yapapi/golem.py

```diff
@@ -131,6 +137,14 @@ def __init__(
         if not strategy:
             strategy = self._initialize_default_strategy()
 
+        if api_config is None:
+            # load deprecate app_key
+            build_dict = {}
+            if app_key is not None:
+                warn_deprecated("app_key", "api_config", "ToDo ver", Deprecated.parameter)  # ToDo
```

Comment:
what will be the next version, what should I put here? `0.11` maybe @shadeofblue

Reply:
yes, 0.11 it would be @lucekdudek